src/main/java/Duke.java

```diff
@@ -1,10 +1,166 @@
+                        System.out.println(messageTask + todoTask.toString() + "\n"
+                                        + "Now you have " + noOfTask + " tasks in the list.");
+
+                    } catch (StringIndexOutOfBoundsException e) {
+                        System.out.println("☹ OOPS!!! The description of a todo cannot be empty.");
+                    }
+    
+    
+                } else if (checkType[0].equals("deadline")) { //check deadline
+    
+                    String deadlineCondition = "/by ";
+
+                    try {
+                        int indexOfTime = input.indexOf(deadlineCondition); //to find /
+                        String dateTime = input.substring(indexOfTime + deadlineCondition.length(), input.length()); // the date and time for by
+                        //convert to the correct one
+                        LocalDateTime deadlineTime = LocalDateTime.parse(dateTime, DateTimeFormatter.ofPattern("d/M/y Hmm"));
+                        String convertedTime = deadlineTime.format(DateTimeFormatter.ofPattern("MMM d yyyy hh:mm a"));
+                        String stringSliced = input.substring(9, indexOfTime); // after deadline
+                        stringArray.add(stringSliced);
+                        Deadline deadlineTask = new Deadline(stringSliced, convertedTime);
+                        taskArray.add(deadlineTask);
+                        String noOfTask = String.valueOf(taskArray.size());
+                        System.out.println(messageTask + deadlineTask.toString() + "\n"
+                                            + "Now you have " + noOfTask + " tasks in the list.");
+                    } catch (StringIndexOutOfBoundsException e) {
+                        System.out.println("☹ OOPS!!! The description of a deadline cannot be empty.");
+                    }
+    
+                } else if (checkType[0].equals("event")) { // check event
+    
+                    String eventCondition = "/at "; 
+                    try {
+                        int indexOfTime = input.indexOf(eventCondition); //to find /
+                        String dateTime = input.substring(indexOfTime + eventCondition.length(), input.length()); // the date and time for by
+                        //convert to the correct one
+                        LocalDateTime eventTime = LocalDateTime.parse(dateTime, DateTimeFormatter.ofPattern("d/M/y Hmm"));
+                        String convertedTime = eventTime.format(DateTimeFormatter.ofPattern("MMM d yyyy hh:mm a"));
+                        String stringSliced = input.substring(6, indexOfTime); // after deadline
+                        stringArray.add(stringSliced);
+                        Event eventTask = new Event(stringSliced, convertedTime);
+                        taskArray.add(eventTask);
+                        String noOfTask = String.valueOf(taskArray.size());
+                        System.out.println(messageTask + eventTask.toString() + "\n"
```

Comment:
Great use of comments to clarify what each line is doing!  Maybe you could also use a few blank lines to separate the code block into logical sections (such as formatting, processing and output.) The coding standard recommends that "Logical units within a block should be separated by one blank line."